driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
Right,  I just need a way of skipping this particular test since 1.2.x does not work well in CCM on windows.  I sort of agree with what you are saying though, should I find an alternative way for skipping this particular test on windows?